src/main/java/gov/cabinetofice/gapuserservice/exceptions/PrivateKeyParsingException.java

```diff
@@ -0,0 +1,14 @@
+package gov.cabinetofice.gapuserservice.exceptions;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(HttpStatus.FORBIDDEN)
```

Comment:
If it's a private key parsing error on the server, is 403 the best response?

Reply:
Yeah fair point, changed to internal server error.